crates/uv-distribution/src/metadata/requires_dist.rs

```diff
@@ -190,6 +217,49 @@ impl From<Metadata> for RequiresDist {
     }
 }
 
+fn apply_source_strategy(
```

Comment:
I was somewhat surprised that this is only used once. I guess the other "dependency types" (like production) don't match this exact pattern?

Reply:
Yeah I actually moved it out because I thought I would need to apply it twice, like.. to the `dev-dependencies` group but I ended up structuring things differently. I think we can move it back in, or make it an impl method somewhere.